src/hooks/useOnchain.tsx

```diff
@@ -54,24 +65,24 @@ function useContractQuery(
 }
 
 export function OnchainProvider({ children }: PropsWithChildren) {
-  const walletClientQuery = useWalletClient({
-    chainId: appConfig.chains[0].id,
-    config: appConfig,
-  });
-  const walletClient = walletClientQuery.data;
-  const publicClient = usePublicClient({
-    chainId: appConfig.chains[0].id,
-    config: appConfig,
+  const { client } = useSmartAccountClient({
+    type: 'LightAccount',
+    // TODO: wtf is the bug that's not using this from config??
```

Comment:
yea i need to figure this out, but for w/e reason i need to pass this explicitly here or else is doesn't get used

Reply:
lol